### PR TITLE
ajout du fichier yaml

### PR DESCRIPTION
**Pull request by jimoussa34**
adding a first branch

### CODE REVIEW

**Comment by userlerueda** [comment]
@jimoussa34 your _pages/jimoussa34.yaml file has some errors, it should be something like this:
`---
githubHandle: jimoussa34
pageUrl: jimoussa34.github.io
timestamp: 2017-01-13
---`

At the moment the CI job is failing and is not allowing you to continue.

**Comment by userlerueda** [comment]
@jimoussa34 the file now looks much better but it will fail again, please press the return key at the end of line 5 so that the red return arrow appears as if it is on line 6.

**Comment by jquiro12** [comment]
Try.

---
githubHandle: jimoussa34
pageUrl: <Pon to pagina web, ejemplo> jimoussa34 .github.io
timestamp: 2017-01-13
---

File: jimoussa34-add-page.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: jimoussa34
+pageUrl: IbnMoussaJaouad
```
Comment (hectorsector) [inline jimoussa34-add-page.yaml]: @jimoussa34 you'll need your web site's full URL here

File: jimoussa34-add-page.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+timestamp: TODAYS-DATE-2017-01-13
```
Comment (hectorsector) [inline jimoussa34-add-page.yaml]: @jimoussa34 you'll also need to remove `TODAYS-DATE-` and just leave `2017-01-13`

**Comment by brianamarie** [comment]
Hi @jimoussa34! Since the URL isn't valid, I'm going to close this PR. Please feel free to reopen if you'd like to continue work!

**Pull request closed without merging**